From 87281f82d484b95e6f809b20f2c68e46d6180216 Mon Sep 17 00:00:00 2001
From: Kent Russell <kent.russell@amd.com>
Date: Fri, 17 Feb 2017 07:49:08 -0500
Subject: [PATCH 1596/4131] drm/amdgpu: Don't use buffer immediately in
 copy_buffer

This was introduced to fix some bugs with BOs using VRAM_CLEARED. We
don't want to use it immediately, so set it to false.

Change-Id: Ie8c907627e12aa1d20922f15b556bd5f71620ff3
Signed-off-by: Kent Russell <kent.russell@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c
index 50af53f..7421062 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c
@@ -2323,7 +2323,7 @@ int amdgpu_amdkfd_copy_mem_to_mem(struct kgd_dev *kgd, struct kgd_mem *src_mem,
 	}
 
 	r = amdgpu_copy_buffer(ring, src_start, dst_start,
-		size, NULL, f);
+		size, NULL, f, false);
 	if (r)
 		goto copy_fail;
 
-- 
2.7.4

